### PR TITLE
Canbus : minor fix bug of and lint, also sync from master

### PR DESCRIPTION
**Pull request by Capri2014**

### CODE REVIEW

File: modules/canbus/tools/teleop.cc [hunk 1]
```diff
@@ -139,7 +139,8 @@ class Teleop {
         perror("read():");
         exit(-1);
       }
-      printf("control command [%s]", control_command_.ShortDebugString().c_str());
+      printf("control command [%s]",
```
Comment (startcode) [inline modules/canbus/tools/teleop.cc]: Use standard LOG functions

File: modules/canbus/tools/teleop.cc [hunk 2]
```diff
@@ -246,14 +247,17 @@ class Teleop {
           // printf("%X\n", c);
           break;
       }
-      printf("control command after switch [%s].", control_command_.ShortDebugString().c_str());
+      printf("control command after switch [%s].",
```
Comment (startcode) [inline modules/canbus/tools/teleop.cc]: use apollo standard LOG functions

File: modules/canbus/tools/teleop.cc [hunk 3]
```diff
@@ -246,14 +247,17 @@ class Teleop {
...
+             control_command_.ShortDebugString().c_str());
     }  // keyboard_loop big while
     tcsetattr(kfd_, TCSANOW, &cooked_);
     printf("keyboard_loop thread quited.\n");
     return;
   }  // end of keyboard loop thread
 
-  ControlCommand &control_command() { return control_command_; }
+  ControlCommand &control_command() {
```
Comment (startcode) [inline modules/canbus/tools/teleop.cc]: reformat with -style=Google

File: modules/canbus/vehicle/gem/protocol/date_time_rpt_83.cc [hunk 4]
```diff
@@ -31,34 +31,48 @@ Datetimerpt83::Datetimerpt83() {}
...
 
-// config detail: {'name': 'time_minute', 'offset': 0.0, 'precision': 1.0, 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 39, 'type': 'int', 'order': 'motorola', 'physical_unit': 'min'}
-int Datetimerpt83::time_minute(const std::uint8_t* bytes, int32_t length) const {
+// config detail: {'name': 'time_minute', 'offset': 0.0, 'precision': 1.0,
+// 'len': 8, 'is_signed_var': False, 'physical_range': '[0|60]', 'bit': 39,
+// 'type': 'int', 'order': 'motorola', 'physical_unit': 'min'}
+int Datetimerpt83::time_minute(const std::uint8_t* bytes,
+                               int32_t length) const {
   Byte t0(bytes + 4);
   int32_t x = t0.get_byte(0, 8);
```
Comment (startcode) [inline modules/canbus/vehicle/gem/protocol/date_time_rpt_83.cc]: can we change template to return t0.get_byte(0, 8)?

Comment (Capri2014) [reply modules/canbus/vehicle/gem/protocol/date_time_rpt_83.cc]: @startcode Will do

**Pull request merged**